app/models/instance.rb

```diff
@@ -15,6 +15,9 @@ def find_tenant_by_host(host)
     end
   end
 
+  VALID_HOST_NAME_REGEX = /\A[a-z0-9]+([\-\.]{1}[a-z0-9]+)*\z/i
+  validates :host, format: { with: VALID_HOST_NAME_REGEX }, length: { maximum: 255 }
```

Comment:
Alright, such domain name exists.

As it says, it was used for DNS resolution, so I guess there is no website
in real life whose domain name has a trailing dot ?

On Thu, Jan 8, 2015 at 10:37 PM, Joel Low <notifications@github.com> wrote:

> In app/models/instance.rb
> <https://github.com/Coursemology/coursemology2/pull/41#discussion-diff-22654310>
> :
>
> > @@ -15,6 +15,9 @@ def find_tenant_by_host(host)
> >      end
> >    end
> >
> > +  VALID_HOST_NAME_REGEX = /\A[a-z0-9]+([\-\.]{1}[a-z0-9]+)*\z/i
> > +  validates :host, format: { with: VALID_HOST_NAME_REGEX }, length: { maximum: 255 }
>
> http://en.wikipedia.org/wiki/Fully_qualified_domain_name my guess is
> correct.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Coursemology/coursemology2/pull/41/files#r22654310>.
>